### ISSUE TITLE
Cannot Build Objective-C Module in Xcode 7.3.1

### ISSUE THREAD

**Issue by otymartin** [ParsePlatform/Parse-SDK-iOS-OSX#919]
@nlutsenko I realize there is an existing thread on this topic but for previous Xcode versions. You linked to a cocoapods issues thread whos solutions where too general purpose for me to follow unfortunately.

It appears everytime I open Xcode, its by pure chance whether the Parse SDK will work or not. Yesterday was fine. Today `Could not build Objective-C module Parse`.
Tomorrow it might work again(frustrating)

My podfile
```swift
# Uncomment this line to define a global platform for your project
platform :ios, '9.0'
# Uncomment this line if you're using Swift
use_frameworks!

target 'ProjectName' do
pod 'Parse'
pod 'ParseLiveQuery'
pod 'Stripe'
pod 'ChameleonFramework'
pod 'SwiftValidator', :git => 'https://github.com/jpotts18/SwiftValidator.git', :branch => 'master'
pod 'EZAlertController'
pod 'OneSignal'
pod 'CardIO'
pod 'Fabric'
pod 'Crashlytics'
pod 'SwiftyJSON'
pod 'SwiftDate'
end
```

**Comment by nlutsenko** [comment]
Sounds pretty frustrating, indeed. Let's try figuring out what's going on.
Is there any other error except of `Could not build Objective-C module Parse` that shows up?
As a general rule with all of these - my first suggestion is to completely delete Xcode's DerivedData, also remove it from Trash (it would require you to quit Xcode when you do it usually), as it uses module cache from there.

Since this is flaky, I would imagine there is something related to order of it caching modules and sometimes it works fine, sometimes it completely busts it...

I am going to look into this specific list of pods together in a bit, maybe I am going to uncover something else here.

P.S. Are you running latest `CocoaPods 1.0.0` that shipped this week? If not - could you try it?

**Comment by otymartin** [comment]
Its the only error.
Another note, I have no bridging header - I've been under the assumption that `use_frameworks!` handles that.  

Yes I just updated to latest Cocoapods

Update:
I deleted `DerivedData` and after a while of processing files and building project.
Got a `No such module` error for a second
Macbook froze
Restarted
Opened Project and rebuild
It worked.

I just had to document the event for people who come across this issue.

Thanks @nlutsenko
